.github/workflows/devcontainer-feature-test.yaml

```diff
@@ -0,0 +1,60 @@
+name: "Test dev container feature"
+on:
+  workflow_dispatch:
+
+  push:
+    branches:
+      - main
+  pull_request:
+    paths:
+      - "ext/devcontainer/**"
+      - ".github/workflows/devcontainer-feature-test.yaml"
+    branches:
+      - main
+  workflow_call:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    continue-on-error: true
+    strategy:
+      matrix:
+        baseImage:
+          - mcr.microsoft.com/devcontainers/base:ubuntu
+          - mcr.microsoft.com/devcontainers/base:debian
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: "Install latest devcontainer CLI"
+        run: npm install -g @devcontainers/cli
+
+      - name: "Generating tests for Radius CLI (default) against '${{ matrix.baseImage }}'"
+        run: devcontainer features test --skip-scenarios -f radcli -i ${{ matrix.baseImage }} .
+        working-directory: ./ext/devcontainer
+
+  test-scenarios:
+    runs-on: ubuntu-latest
+    continue-on-error: true
+    
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: "Install latest devcontainer CLI"
+        run: npm install -g @devcontainers/cli
+
+      - name: "Generating tests for radcli scenarios"
+        run: devcontainer features test -f radcli --skip-autogenerated --skip-duplicated .
+        working-directory: ./ext/devcontainer
+
+  test-global:
+    runs-on: ubuntu-latest
+    continue-on-error: true
```

Comment:
Do you know why this is using `continue-on-error` so liberally? I'd want this to fail if the tests fail.

Reply:
I have to check, because I saw failures of the run despite this setting when I did my first tries in the local repo. I will check and report back and in case  adjust the PR accordingly

Reply:
Checked it in my playground repo. This is indeed confusing. I removed the continue-on-error as this clearly shows if something failed on the overview page. Otherwise you must navigate into the run.

I also removed the `test-global` section as we do not have mutiple features that we want to test in one run, but can keep it simple with the feature-specific test parts

Reply:
Thanks, I feel better about seeing this without `continue-on-error`.